.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: Pre-commit checks
+
+on:
+  pull_request:
```

Comment:
Should we not define which branch(es) this runs against?
I would expect to see:
```
on:
  pull_request:
    branches:
      - master
 ```

Reply:
yes, we need it. thank you!